File: src/main/java/com/nunchuk/android/type/GroupSpendingLimitIntervalHelper.java
package com.nunchuk.android.type;

public final class GroupSpendingLimitIntervalHelper {

    private GroupSpendingLimitIntervalHelper() {
    }

    public static GroupSpendingLimitInterval from(final int ordinal) {
        return CollectionUtil.fromOrdinal(GroupSpendingLimitInterval.values(),
                ordinal,
                GroupSpendingLimitInterval.DAILY
        );
    }

}

File: src/main/java/com/nunchuk/android/type/GroupSpendingLimitIntervalHelper.kt
package com.nunchuk.android.type

object GroupSpendingLimitIntervalHelper {
    @JvmStatic
    fun from(ordinal: Int): GroupSpendingLimitInterval =
        CollectionUtil.fromOrdinal(GroupSpendingLimitInterval.entries, ordinal, GroupSpendingLimitInterval.DAILY)
}
